docs/content/migration/v2-to-v3.md

```diff
@@ -147,6 +147,7 @@ It is now unsupported and would prevent Traefik to start.
 ##### Remediation
 
 The `http3` option should be removed from the static configuration experimental section.
+To enable `http3` again, see [entrypoint configuration](https://doc.traefik.io/traefik/v3.0/routing/entrypoints/#http3_1).
```

Comment:
Hey @yyewolf,

Thank you for your contribution.
However, configuring HTTP/3 on the Entrypoints was already required in v2, the experimental flag allowed to enable the feature and configure it.
For such a reason, I propose the following modification:
```suggestion
To configure `http3, see [entrypoint configuration](https://doc.traefik.io/traefik/v3.0/routing/entrypoints/#http3_1).
```